runtime/hookable/hookableshared.mk.ftl

```diff
@@ -61,6 +61,10 @@ EXPORT_FUNCTIONS_FILE := $(HOOKABLE_SRCDIR)hookable.exportlist
 MODULE_STATIC_LIBS += j9utilcore j9pool
 MODULE_SHARED_LIBS += j9thr${uma.buildinfo.version.major}${uma.buildinfo.version.minor}
 
+ifeq (osx,$(OMR_HOST_OS))
+GLOBAL_LDFLAGS += -compatibility_version 1.0.0 -current_version 1.0.0
+endif # OSX
+
```

Comment:
This change will not be necessary once eclipse/omr#5251 lands.

Reply:
Remove it.